### ISSUE TITLE
Ctrl-S for triggering draft save

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#634]
Currently, pressing Ctrl-S when you are writing an email brings up the »Save Document as« dialog which is not really helpful.

Instead, it should force trigger a draft save if you are in any of the compose or reply fields. (Maybe we should even catch it in all cases, so that the dialog never pops up.)

**Comment by AasthaGupta** [comment]
I would like to work on this.

**Comment by ChristophWurst** [comment]
@AasthaGupta awesome! Let me know if you need any help

**Comment by ChristophWurst** [comment]
> @jancborchardt I'd like to work on this.

Hey, @AasthaGupta! Are you still interested in working on this?

**Comment by StCyr** [comment]
function `handleShortcuts()` in `EnvelopeList.vue` shows how to do that

**Comment by anestv** [comment]
This seems easy, but more importantly is it still relevant? Since PR #1236 the draft seems to be saved on every key press

**Comment by ChristophWurst** [comment]
Yes but still it would be nice to catch it. Otherwise you get the default *save page* dialog.
